scripts/asm-installer/install_asm

```diff
@@ -2084,8 +2084,10 @@ install_managed_components() {
   URL="$(kubectl get mutatingwebhookconfiguration istiod-asm-managed -ojson | jq .webhooks[0].clientConfig.url -r)"
   # shellcheck disable=SC2001
   URL="$(echo "${URL}" | sed 's/inject.*$/validate/g')"
+  CLOUDRUN_ADDR=$(kubectl get mutatingwebhookconfigurations istiod-asm-managed -ojson | jq ".webhooks[0].clientConfig.url" | cut -d'/' -f3)
 
   run kpt cfg set asm anthos.servicemesh.controlplane.validation-url "${URL}"
+  run kpt cfg set asm anthos.servicemesh.managed-controlplane.cloudrun-addr "${CLOUDRUN_ADDR}"
```

Comment:
The first $URL already has this, change the second URL to VALIDATION_URL and then have CLOUDRUN_ADDR use $URL instead of querying the cluster again

Reply:
Done.